tests/app/test_commands.py

```diff
@@ -0,0 +1,24 @@
+from app.commands import local_dev_broadcast_permissions
+from app.dao.services_dao import dao_add_user_to_service
+from tests.app.db import create_user
+
+
+def test_local_dev_broadcast_permissions(
+    sample_service,
+    sample_broadcast_service,
+    notify_api,
+    mocker,
```

Comment:
`mocker` is not used in this test

Reply:
Good spot - I've amended the second commit to remove it.